<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temel AI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1861</width>
      <height>911</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>11</number>
    </property>
    <widget class="QWidget" name="TEMEL">
     <attribute name="title">
      <string>TEMEL</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Book</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>PUA</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Movie</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Music</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>My Projects</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Health</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Country</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1131</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Picture</string>
      </property>
      <widget class="QLineEdit" name="image_textbox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Keywords</string>
       </property>
      </widget>
      <widget class="QComboBox" name="picture_size">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>170</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Min Size (KB)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_image_button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="download_number">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>300</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Downloaded</string>
       </property>
      </widget>
      <widget class="QLabel" name="downloaded_label">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>551</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>1131</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Youtube</string>
      </property>
      <widget class="QLineEdit" name="image_textbox_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Keywords</string>
       </property>
      </widget>
      <widget class="QComboBox" name="picture_size_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>170</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Min Size (KB)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_youtube_video_button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="download_number_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>300</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Downloaded</string>
       </property>
      </widget>
      <widget class="QLabel" name="downloaded_label_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Social Media</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Instagram</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QLineEdit" name="instagram_username_textbox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QPushButton" name="instagram_search_button">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Followers :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Following :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Post :</string>
       </property>
      </widget>
      <widget class="QLabel" name="post_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>####</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>####</string>
       </property>
      </widget>
      <widget class="QLabel" name="following_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>####</string>
       </property>
      </widget>
      <widget class="QLabel" name="followers_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>####</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Web Browser</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>1811</width>
        <height>621</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>English</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>80</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>72</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>English :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>69</y>
        <width>851</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Turkce :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>120</y>
        <width>851</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>380</y>
        <width>1201</width>
        <height>441</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>600</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>39</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Turkish</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>English</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="showAllButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>832</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show all</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
